qiskit_nature/circuit/library/initial_states/hartree_fock.py

```diff
@@ -46,8 +46,9 @@ def __init__(
         """
         if isinstance(qubit_converter.mapper, BravyiKitaevSuperFastMapper):
             raise TypeError(
-                "Unsupported mapper in qubit_converter: ", type(qubit_converter.mapper),
-                ". See issue 436."
+                "Unsupported mapper in qubit_converter: ",
+                type(qubit_converter.mapper),
+                ". See issue 436.",
```

Comment:
I suggest changing this to the entire URL. With this number not everyone may know where to find this.
A URL is also very convenient to open directly from the terminal.